src/pkg/util/uri/uri_test.go

```diff
@@ -35,10 +35,13 @@ func Test_SplitURI(t *testing.T) {
 		transport string
 		ref       string
 	}{
-		{"docker no scope", "docker://ubuntu", "docker", "//ubuntu"},
+		{"docker basic", "docker://ubuntu", "docker", "//ubuntu"},
 		{"docker scoped", "docker://user/image", "docker", "//user/image"},
 		{"dave's magical lolcow", "docker://godlovedc/lolcow", "docker", "//godlovedc/lolcow"},
-		{"docker w/ tags", "docker://godlovedc/lolcow:latest", "docker", "//godlovedc/lolcow:latest"},
+		{"docker with tags", "docker://godlovedc/lolcow:latest", "docker", "//godlovedc/lolcow:latest"},
+		{"library basic", "library://image", "library", "//image"},
+		{"library scoped", "library://collection/image", "library", "//collection/image"},
```

Comment:
Technically library can be:

`library://container` or `library://container:tag` (using default entity/collection)
`library://collection/container` or `library://collection/container:tag` (using default entity)
`library://entity/collection/container` or `library://entity/collection/container:tag` (full path)